assets/js/atomic/blocks/product-elements/price-v2/block.json

```diff
@@ -0,0 +1,49 @@
+{
+	"$schema": "http://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-price-v2",
+	"title": "Price",
+	"keywords": "WooCommerce",
+	"category": "woocommerce-product-elements",
+	"textdomain": "woo-gutenberg-products-block",
+	"description": "Display the price of a product, including any discounts.",
+	"supports": {
+		"layout": {
+			"allowSwitching": false,
+			"allowInheriting": false,
+			"default": { "type": "flex" }
+		},
```

Comment:
Note, the `layout` supports property only became recently available and will surface in WordPress 6.3. Prior to that it was `__experimentalLayout` which is still supported. It may be better to ship with `__experimentalLayout if we want to support WP versions < 6.3 but even then testing should cover which versions of WP this will explicitly be supported by as that may influence what versions of WP this block is surfaced for.